pyproject.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "^3.7"
-Django = "^3.2"
+Django = "^3.2.17"
```

Comment:
Suggest not specifying patch version here as overly precise and won't get bug fixes when do `poetry update`.  NB assumes Django project uses semantic versioning!

Reply:
Ta! How does this guarantee a properly patched version of Django in deployment?

Reply:
If using a comprehensive CI/CD pipeline then the CD could redeploy following the merging of fixes prompted by dependabot.  Otherwise, tools such as `pip audit` or `safety` can scan for vuln pkgs and that could be a prompt for (a less automated) redeployment.

Reply:
So often a `poetry update` then redeployment is all that's needed.